notional-labs/ibc-go
# test diff of simapp-refactor and 50

- sdk 50 first go
- add go workspace
- make proto-gen, cleanup
- additiona cleanup of imports
- updates...
- handle expedited proposal bools
- restructure modules
- pre revert commit
- Revert "restructure modules"
- correctly migrate modules
- migrate parameter store
- fix calls to NewContext
- tidy and enable thelper
- use thelper and disable depguard
- use latest golangci-lint in ci
- upgrade to go 1.20 and newer protobuf builder to reduce v50 changeset
- more cleanup, 5% of tests pass
- add go work
- bumps
- bump go in an additional place
- set up rootmultistore in the new way and address WrongArgCount lints
- app.go and queries
- chain.go
- query command in root.go
- context should go first
- additional cleanup, mainly changing method signatures
- update message signatures and use sdk main branch
- transfer authorization changes
- move go.work.example back
- bump go version
- use single quotes for the go version
- use sdk 50
- Revert "use single quotes for the go version"
- use single quotes for real
- tidy
- fix
- update .gitignore
- tidies again
- fixing
- register modulebasicmanager in app.go
- add module register in app.go
- fix the upgrade module's return
- all errors now checked
- revert changes to the capability module path
- add circuit breaker
- Begin to use modulebasicmanager
- rename Marshaler to Codec
- account for modulebasicmanager
- fix setup()
- fix import cycle
- use appcodec for setuptestingapp
- add missing IBCKeeper initialize and remove unused module from module Manager
- update SetupWithGenesisValSet func
- sdk 50 first go
- add go workspace
- make proto-gen, cleanup
- additiona cleanup of imports
- updates...
- handle expedited proposal bools
- restructure modules
- pre revert commit
- Revert "restructure modules"
- correctly migrate modules
- migrate parameter store
- fix calls to NewContext
- tidy and enable thelper
- use thelper and disable depguard
- use latest golangci-lint in ci
- upgrade to go 1.20 and newer protobuf builder to reduce v50 changeset
- more cleanup, 5% of tests pass
- add go work
- bumps
- set up rootmultistore in the new way and address WrongArgCount lints
- fixing
- Merge branch 'sdk-50' of https://github.com/notional-labs/ibc-go into sdk-50
- register modulebasicmanager in app.go
- add module register in app.go
- Merge branch 'sdk-50' of https://github.com/notional-labs/ibc-go into sdk-50
- fix the upgrade module's return
- all errors now checked
- update FinalizeBlock height input
- update FinalizeBlock height input
- revert changes to the capability module path
- add circuit breaker
- Begin to use modulebasicmanager
- rename Marshaler to Codec
- account for modulebasicmanager
- fix setup()
- fix import cycle
- use appcodec for setuptestingapp
- add missing IBCKeeper initialize and remove unused module from module Manager
- update SetupWithGenesisValSet func
- fixups
- remove argument from NewContext
- remove unused code
- refactor FundAccount usage
- fix test
- coins.IsEqual -> coins.Equal
- replace errorsmod path
- fix SetupTest
- remove unused variable
- change function origin
- sdk -> storetypes
- replace name
- cleanup app.go like goland diff ~/ibc-go/testing/simapp ~/cosmos-sdk/simapp
- ModuleManager
- add makeCommit func
- fixing
- minor
- remove double imports
- linting
- add interface registry for ibc light client
- register client state
- bump cometbft to the latest commit from the v0.38.x branch
- parallel tests
- parallelize one more test
- undeprecate stuff
- unwrap sdk context: deprecated
- big ctx unwrapping
- accountI
- undeprecate events
- amino
- 29-fee amino
- add interface registry solomachine
- change to legacy context
- add dockerfile
- eliminate changes to .github
- Revert "parallel tests"
- make some of the changes suggested by @alpe / review wasmd's implementation of code similar to testing/chain.go
- push changes to chain.go w/ attribution
- correctly eliminate changes from .github
- remove the compatiblity test matrices
- remove all of .github
- re-add .github checked out from the feature branch
- revert gitignore changes
- remove .dockerignore
- use main's linter
- eliminate changes vs main in .github
- eliminate changes in main vs docs
